base/dcp_client.go

```diff
@@ -49,6 +49,7 @@ type DCPClientOptions struct {
 	FailOnRollback             bool           // When true, the DCP client will terminate on DCP rollback
 	InitialMetadata            []DCPMetadata  // When set, will be used as initial metadata for the DCP feed.  Will override any persisted metadata
 	CheckpointPersistFrequency *time.Duration // Overrides metadata persistence frequency - intended for test use
+	MetadataType               int            // Overrides metadata persistence frequency - intended for test use
```

Comment:
- might be nice to have a type alias like `cbgtFeedType`
- comment needs updating

Reply:
Updated comments.